Source/WebCore/PAL/pal/spi/mac/NSSharingServicePickerSPI.h

```diff
@@ -33,6 +33,7 @@ DECLARE_SYSTEM_HEADER
 
 @interface NSSharingServicePicker (Staging_r88855017)
 - (NSMenuItem *)standardShareMenuItem;
+- (NSMenuItem *)standardShareMenuItemRelativeToRect:(NSRect)rect ofView:(NSView *)view preferredEdge:(NSRectEdge)preferredEdge;
 @end
```

Comment:
This category can be deleted, `NSSharingService_Private.h` contains the declarations.